velox/functions/sparksql/aggregates/SumAggregate.cpp

```diff
@@ -38,6 +63,15 @@ exec::AggregateRegistrationResult registerSum(const std::string& name) {
           .intermediateType("double")
           .argumentType("double")
           .build(),
+      exec::AggregateFunctionSignatureBuilder()
+          .integerVariable("a_precision")
+          .integerVariable("a_scale")
+          .integerVariable("r_precision", "min(38, a_precision + 10)")
```

Comment:
Just curious, why a_precision + 10 here?

Reply:
This is the definition of Spark, you can check it [here](https://github.com/apache/spark/blob/5b2da4865622e3d8481a8a74f483c9de31e03268/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/Sum.scala#L77).